browser/resources/settings/brave_clear_browsing_data_dialog/brave_clear_browsing_data_dialog_behavior.ts

```diff
@@ -1,151 +1,163 @@
-      this.onSelectedPageChangedCallback_);
+      this.onSelectedPageChangedCallback_)
 
-    this.updateSaveButtonStateCallback_ = this.updateSaveButtonState_.bind(this);
-    this.shadowRoot.querySelector('#on-exit-tab').addEventListener(
-      'clear-data-on-exit-page-change', this.updateSaveButtonStateCallback_);
+    this.updateSaveButtonStateCallback_ = this.updateSaveButtonState_.bind(this)
+    this.shadowRoot!.querySelector('#on-exit-tab')!.addEventListener(
+      'clear-data-on-exit-page-change', this.updateSaveButtonStateCallback_)
 
-    this.saveOnExitSettingsCallback_ = this.saveOnExitSettings_.bind(this);
-    this.shadowRoot.querySelector('#saveOnExitSettingsConfirm').addEventListener(
-      'click', this.saveOnExitSettingsCallback_);
+    this.saveOnExitSettingsCallback_ = this.saveOnExitSettings_.bind(this)
+    this.shadowRoot!.querySelector('#saveOnExitSettingsConfirm')!
+      .addEventListener('click', this.saveOnExitSettingsCallback_)
 
-    this.onClearBraveAdsDataClickHandler_ = this.clearBraveAdsData_.bind(this);
+    this.onClearBraveAdsDataClickHandler_ = this.clearBraveAdsData_.bind(this)
   }
 
   override disconnectedCallback() {
     super.disconnectedCallback()
 
     if (this.onSelectedPageChangedCallback_) {
       this.$.pages.removeEventListener('selected-item-changed',
-        this.onSelectedPageChangedCallback_);
-      this.onSelectedPageChangedCallback_ = null;
+        this.onSelectedPageChangedCallback_)
+      this.onSelectedPageChangedCallback_ = null
     }
 
-    if (this.saveOnExitSettingsCallback_) {
-      this.shadowRoot.querySelector('#on-exit-tab').removeEventListener(
-        'clear-data-on-exit-page-change', this.updateSaveButtonStateCallback_);
-      this.updateSaveButtonStateCallback_ = null;
+    if (this.updateSaveButtonStateCallback_) {
+      this.shadowRoot!.querySelector('#on-exit-tab')!.removeEventListener(
+        'clear-data-on-exit-page-change', this.updateSaveButtonStateCallback_)
+      this.updateSaveButtonStateCallback_ = null
     }
 
     if (this.saveOnExitSettingsCallback_) {
-      this.shadowRoot.querySelector('#saveOnExitSettingsConfirm').removeEventListener(
-        'click', this.saveOnExitSettingsCallback_);
-      this.saveOnExitSettingsCallback_ = null;
+      this.shadowRoot!.
+        querySelector('#saveOnExitSettingsConfirm')!.
+        removeEventListener('click', this.saveOnExitSettingsCallback_)
+      this.saveOnExitSettingsCallback_ = null
     }
 
-    this.onClearBraveAdsDataClickHandler_ = () => {};
+    this.onClearBraveAdsDataClickHandler_ = () => {}
   }
 
 /**
   * Updates the text of a browsing data counter corresponding to the given
   * preference.
   * @param {string} prefName Browsing data type deletion preference.
   * @param {string} text The text with which to update the counter
-  * @private
   */
-  updateOnExitCountersText(prefName, text) {
+  private updateOnExitCountersText(prefName: string, text: string) {
```

Comment:
nit: should this be
```suggestion
  private updateOnExitCountersText_(prefName: string, text: string) {
```

Reply:
Fixed here a2867cd1721f5a3bdcf9e28e87c548bb24b8b407, thanks!